trollmoves/movers.py

```diff
@@ -42,6 +43,12 @@
 
 LOGGER = logging.getLogger(__name__)
 
+HOME = os.environ.get('HOME')
+if HOME:
+    NETRCFILE = os.path.join(HOME, '.netrc')
+else:
+    NETRCFILE = None
+
```

Comment:
This isn't needed. According to the documentation of netrc, the filename is optional and defaults to home's .netrc: https://docs.python.org/3.8/library/netrc.html#netrc.netrc

Reply:
Well, you are right. However, this came from experience with another application where HOME is not defined. As you know when run from supervisor HOME is not available. Therefore I implemented this. If home is not set you will get the following in your case:
```
In [5]: secrets = netrc.netrc()                                                                                                                                               
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
<ipython-input-5-211a7aaf8c74> in <module>
----> 1 secrets = netrc.netrc()

~home/a000680/miniconda3/envs/python38/lib/python3.8/netrc.py in __init__(self, file)
     27         self.hosts = {}
     28         self.macros = {}
---> 29         with open(file) as fp:
     30             self._parse(file, fp, default_netrc)
     31 

FileNotFoundError: [Errno 2] No such file or directory: '/.netrc'
```
@mraspaud Do you suggest I catch that exception then and log a warning?

Reply:
if `$HOME` isn't available, the `netrc` module will surely raise an exception, why not just catch it ?

Reply:
So the answer to your question is 'yes'.

Reply:
also note that is the `.netrc` file isn't present in `$HOME`, you'll have to catch a `FileNotFoundError`

Reply:
@mraspaud Can you please give me a hint where you would have such a small test? I find no tests for movers.py, so I suppose it somewhere in test_dispatcher.py? These tests seem to test on a higher level, or?

Reply:
I didn't realise there wasn't any test for movers. Could you create a test file for movers and place just a simple test there for the ftpmover's change you made ? If the file is there, we can then populate it with more tests